CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.7.3 – 2025-04-03
+
+1. Add more information to `$feature_flag_called` events for `/decide` requests such as flag id, version, reason, and the request id.
```

Comment:
worth a minor bump vs a patch?  Can't remember what we've done for the other repos.